src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+
+/**
+ * Contains the executables when the user uses the 'bye' command.
+ */
```

Comment:
Perhaps you could include the @author tag for your description of the classes. Otherwise i think the description is quite clear!

Reply:
Yep, ill prolly add it in. Thanks :D